From cbc0075a13e9fb86069c4979ba7b6bf5885b4706 Mon Sep 17 00:00:00 2001
From: Xinhuang Li <buluess.li@rock-chips.com>
Date: Thu, 30 Nov 2017 10:19:21 +0800
Subject: [PATCH] input: rc: Fix clk get error

when the work clock is the same as the bus clock.drop
the "clock-names" property as the PWM uses only one
clock (the driver now needs to call devm_clk_get with NULL).

Change-Id: I7fcdd9635f81778a3dc522d99d0fc081408d6fea
Signed-off-by: Xinhuang Li <buluess.li@rock-chips.com>
---
 drivers/input/remotectl/rockchip_pwm_remotectl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/remotectl/rockchip_pwm_remotectl.c b/drivers/input/remotectl/rockchip_pwm_remotectl.c
index f67b704ca4ac..dd0faae58be5 100644
--- a/drivers/input/remotectl/rockchip_pwm_remotectl.c
+++ b/drivers/input/remotectl/rockchip_pwm_remotectl.c
@@ -425,7 +425,7 @@ static int rk_pwm_probe(struct platform_device *pdev)
 		clk = devm_clk_get(&pdev->dev, "pwm");
 		p_clk = devm_clk_get(&pdev->dev, "pclk");
 	} else {
-		clk = devm_clk_get(&pdev->dev, "pclk_pwm");
+		clk = devm_clk_get(&pdev->dev, NULL);
 		p_clk = clk;
 	}
 	if (IS_ERR(clk)) {
-- 
2.35.3

